firmware/signing.c

```diff
@@ -1025,7 +1079,6 @@ void signing_txack(TransactionType *tx)
 				tx->inputs[0].script_sig.size = 0;
 			}
 			resp.serialized.serialized_tx.size = tx_serialize_input(&to, &tx->inputs[0], resp.serialized.serialized_tx.bytes);
-			update_ctr = 0;
```

Comment:
This is a minor  bug fix. The update_ctr counts when the gear should be redrawn.   Previously we had redrawn the gears here and reset update_ctr, after moving the redraw code around, this stuck here